web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -1,63 +1,147 @@
-- The minimum version of PyArrow will be bumped every major pandas release to the highest
-  PyArrow version that has been released for at least 2 years, and the minimum PyArrow version will be
-  maintained for every minor version in the major version series.
+- PyArrow becomes a runtime dependency starting with pandas 3.0
+- The minimum version of PyArrow supported starting with pandas 3.0 is version 7 of PyArrow.
+- When the minimum version of PyArrow is bumped, PyArrow will be bumped to the highest version that has
+  been released for at least 2 years.
+- Starting in pandas 2.1, pandas raises a ``FutureWarning`` when needing to infer string data that the future
+  data type result will be `ArrowDtype` with `pyarrow.string` instead of object
+- Starting in pandas 3.0, the default type inferred for string data will be `ArrowDtype` with `pyarrow.string`
+  instead of `object`
 
 ## Background
 
 PyArrow is an optional dependency of pandas that provides a wide range of supplemental features to pandas:
 
 - Since pandas 0.21.0, PyArrow provided I/O reading functionality for Parquet
-- Since pandas 1.2.0, pandas integrated PyArrow into the `ExtensionArray` interface to provide an optional string data type backed by PyArrow
+- Since pandas 1.2.0, pandas integrated PyArrow into the `ExtensionArray` interface to provide an
+  optional string data type backed by PyArrow
 - Since pandas 1.4.0, PyArrow provided I/0 reading functionality for CSV
-- Since pandas 1.5.0, pandas provided an `ArrowExtensionArray` and `ArrowDtype` to support all PyArrow data types within the `ExtensionArray` interface
-- Since pandas 2.0.0, All I/O readers have the option to return PyArrow-backed data types, and many methods now utilize PyArrow compute functions to
+- Since pandas 1.5.0, pandas provided an `ArrowExtensionArray` and `ArrowDtype` to support all PyArrow
+  data types within the `ExtensionArray` interface
+- Since pandas 2.0.0, all I/O readers have the option to return PyArrow-backed data types, and many methods
+  now utilize PyArrow compute functions to
 accelerate PyArrow-backed data in pandas, notibly string and datetime types.
 
 As of pandas 2.0, one can feasibly utilize PyArrow as an alternative data representation to NumPy with advantages such as:
 
-1. Consistent ``NA`` support for all data types
-2. Broader support of data types such as ``decimal``, ``date`` and nested types
+1. Consistent `NA` support for all data types
+2. Broader support of data types such as `decimal`, `date` and nested types
+
+Additionally, when users pass string data into pandas constructors without specifying a data type, the result data type
+is `object`. With pyarrow string support available since 1.2.0, requiring pyarrow for 3.0 will allow pandas to default
+the inferred type to the more efficient pyarrow string type.
+
+```python
+In [1]: import pandas as pd
+
+In [2]: pd.Series(["a"]).dtype
+Out[2]: dtype('O')
+```
```

Comment:
The context of this example is a little confusing.  I think you want to indicate that this is current behavior, and then provide an example of what the proposed new behavior would be as of version 3.0.